lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -654,3 +657,16 @@ func (s *builderServicer) getSyncInterval(nwEpc *lte_models.NetworkEpcConfigs, g
 	}
 	return minSyncInterval
 }
+
+// getRandomizedSyncInterval returns the interval received from getSyncInterval increased by a random
+// delta in the range [0, getSyncInterval() / 5.0]. Increased sync interval ameliorates the thundering
+// herd effect at the orc8r. Delta is deterministic based on gwKey
```

Comment:
Missed this, but let's make sure to mention on the docs or on the variable itself that this is measured in secs

Reply:
Included unit in doc string.